pactman/verifier/pytest_plugin.py

```diff
@@ -14,14 +14,15 @@ def pytest_addoption(parser):
                      help="pact JSON files to verify (wildcards allowed)")
     parser.addoption("--pact-broker-url", default='',
                      help="pact broker URL")
-    parser.addoption("--pact-broker-auth", default='',
-                     help="pact broker basic auth user:password")
     parser.addoption("--pact-broker-token", default='',
                      help="pact broker bearer token")
     parser.addoption("--pact-provider-name", default=None,
                      help="provider pact name")
     parser.addoption("--pact-consumer-name", default=None,
                      help="consumer to limit verification to")
+    parser.addoption("--pact-consumer-version-tag", metavar='TAG', action='append',
+                     help='limit broker pacts tested to those matching the tag. May be specified '
+                          'multiple times to further restrict the set of pacts.')
```

Comment:
I think "May be specified multiple times to further restrict the set of pacts" should be something like "May be specified multiple times in which case pacts matching any of these tags will be verified."